### PR TITLE
feat: structure docs in content directory

### PR DESCRIPTION
**Pull request by saihaj**
### Summary of PR
* [X] Remove old boilerplates docs and blog pages
* [X] Add dummy pages for ShabadOS products
* [X] Add products to main navigation bar
* [X] Add product tiles to homepage

They way I have structured docs is that all the product docs go in `content/`. Make a folder for product. Markdown files need to get processed so we can generate pages, it also needs to know route it should serve docs on. So we fill in `src/pages.config.js` and add route, path, sidebar and other information. This way allows us to customize product docs independently. We can have different layouts and many other features (I can't think of any for now except layout).

For now `docs` directory is not used for anything and I tried researching to remove it but no success yet. I need it because of the base theme to apply.  So I plan to put all the docs for this site and maybe move over our wiki from `.github` repo. Something we can discuss in future what to do. But for now we should ignore this and I will handle it in future PR.

**Deploy preview:** https://shabad-os-docs-git-other-way-strucutre-docs.saihaj.vercel.app/

### Time spent on PR
7 hours - Spent too much time researching and finding an elegant way to have multiple docs and make it easy to just add new products in future.

### Reviewers
@Harjot1Singh @bhajneet

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -110,17 +103,15 @@ module.exports = {
       ],
     },
   },
+  plugins: [
+    ...pages,
```
Comment (Harjot1Singh) [inline docusaurus.config.js:107]: Why spread them?

File: src/pages/index.js [hunk 2]
```diff
@@ -75,32 +64,19 @@ const Home = () => {
...
-            <div className="container">
-              <div className="row">
-                {features.map( props => (
-                  <Feature key={props.title} {...props} />
-                ) )}
-              </div>
+        <section className={styles.features}>
+          <div className="container">
+            <div className="row">
+              { products.map( ( { title, description } ) => (
```
Comment (Harjot1Singh) [inline src/pages/index.js:74]:
```suggestion
              {products.map( ( { title, description } ) => (
```

File: docusaurus.config.js [hunk 3]
```diff
@@ -37,13 +41,19 @@ module.exports = {
         src: 'img/logo.png',
       },
       items: [
-        {
-          to: 'docs/',
-          activeBasePath: 'docs',
-          label: 'Docs',
+        { to: '/',
```
Comment (Harjot1Singh) [inline docusaurus.config.js:44]: lol wut formatting is dis

**Review by Harjot1Singh — CHANGES_REQUESTED** [review]
Aside from the inconsequential stylstic issues highlighted, looks good.

I personally like keeping the content under `content`. I reckon it'd be good for each content folder to have a `sidebar.json` similar to https://github.com/facebook/react-native-website/blob/master/website/sidebars.json

**Comment by saihaj** [comment]
> I personally like keeping the content under content. I reckon it'd be good for each content folder to have a sidebar.json similar to https://github.com/facebook/react-native-website/blob/master/website/sidebars.json

Yes idea is that inside `src/sidebars` we will have sidebars for each directory that is in `content`. This is something my next PR will bring.

File: docusaurus.config.js (hunk 1 above)
Comment (saihaj) [reply docusaurus.config.js:107]: see `src/pages.config.js` the config was getting very big and there was repetition. So I decided to move `pages` plugins to that. So in future one just needs to add this to the `pages` array in `src/pages.config.js`
```
  [
    '@docusaurus/plugin-content-docs',
    {
      ...sharedValues,
      id: 'someCoolProject',
      path: 'content/someCoolProject',
      routeBasePath: 'someCoolProject',
    },
  ],
```

Comment (Harjot1Singh) [reply docusaurus.config.js]: Yeah, that makes sense, but you're doing `plugins: [ ...pages ]` which is the exact same as `plugins: pages`, unless I'm missing something.

Comment (saihaj) [reply docusaurus.config.js]: We can add more plugins so that is why I did spread. But right now there is nothing I can change it

Comment (Harjot1Singh) [reply docusaurus.config.js]: Oh, I see. No problem.

**Review by Harjot1Singh — APPROVED** [review]

**Pull request merged**